public/app/features/admin/Users/AnonUsersTable.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useMemo } from 'react';
+
+import {
+  Avatar,
+  CellProps,
+  Column,
+  FetchDataFunc,
+  InteractiveTable,
+  Pagination,
+  Stack,
+  Badge,
+  Tooltip,
+} from '@grafana/ui';
+import { UserAnonymousDeviceDTO } from 'app/types';
+
+type Cell<T extends keyof UserAnonymousDeviceDTO = keyof UserAnonymousDeviceDTO> = CellProps<
+  UserAnonymousDeviceDTO,
+  UserAnonymousDeviceDTO[T]
+>;
+
+// A helper function to parse the user agent string and extract parts
+const parseUserAgent = (userAgent: string) => {
+  const parts = {
+    browser: userAgent.split(' ')[0],
+    computer: userAgent.split(' ')[1],
+  };
+  return parts;
+};
+
+// A helper function to truncate each part of the user agent
+const truncatePart = (part: string, maxLength: number) => {
+  return part.length > maxLength ? part.substring(0, maxLength) + '...' : part;
+};
+
+interface UserAgentCellProps {
+  value: string;
+}
+
+const UserAgentCell = ({ value }: UserAgentCellProps) => {
+  const parts = parseUserAgent(value);
+  return (
+    <div>
+      <Tooltip theme="info-alt" content={value} placement="top-end" interactive={true}>
+        <span>
+          {truncatePart(parts.browser, 10)}
+          {truncatePart(parts.computer, 10)}
+        </span>
+      </Tooltip>
+    </div>
```

Comment:
```suggestion
  <Tooltip theme="info-alt" content={value} placement="top-end" interactive={true}>
    <span>
      {truncatePart(parts.browser, 10)}
      {truncatePart(parts.computer, 10)}
    </span>
  </Tooltip>
```
An extra div wrapper is no necessary here.